Corona-Warn-App/src/main/java/de/rki/coronawarnapp/http/OfflineCacheInterceptor.kt

```diff
@@ -0,0 +1,48 @@
+package de.rki.coronawarnapp.http
+
+import android.content.Context
+import de.rki.coronawarnapp.util.ConnectivityHelper.isNetworkEnabled
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class OfflineCacheInterceptor(private val context: Context) : Interceptor {
+    companion object {
+        private const val MAX_AGE = 5
+        private const val MAX_STALE_DAYS = 1
+        private const val MAX_STALE = 60 * 60 * 24 * MAX_STALE_DAYS
+    }
+    override fun intercept(chain: Interceptor.Chain): Response {
+        // Get the request from the chain.
+        var request = chain.request()
```

Comment:
OkHttp has a more fluent api for cache control perhaps it's better to use?
https://square.github.io/okhttp/4.x/okhttp/okhttp3/-cache-control/

Reply:
I will check it out and adapt in the next release if we decide to invest.